viam-orb-slam3_test.go

```diff
@@ -72,34 +68,13 @@ func getNumOrbslamImages(mode slam.Mode) int {
 	}
 }
 
-func createFakeSLAMLibraries() {
-	for _, s := range slam.SLAMLibraries {
-		slam.SLAMLibraries["fake_"+s.AlgoName] = slam.LibraryMetadata{
-			AlgoName:       "fake_" + s.AlgoName,
-			AlgoType:       s.AlgoType,
-			SlamMode:       s.SlamMode,
-			BinaryLocation: "true",
-		}
-	}
-}
-
-func deleteFakeSLAMLibraries() {
-	for k := range slam.SLAMLibraries {
-		if strings.Contains(k, "fake") {
-			delete(slam.SLAMLibraries, k)
-		}
-	}
-}
-
```

Comment:
These functions were used to make the tests portable across carto and orb. That is no longer required.

Reply:
Do you mind explaining why before we did not need to save and restore the original binary location, but now we do?

Reply:
Before we did used SLAMLibrary to store the BinaryLocation and then we just pointed all of the tests to run against the SLAMLibrary. That abstraction is being phased out. This was the next best option. We thought about having the New(...) function accept a BinaryLocation but decided against it (without much real reasoning, mostly just intuition). Do you have any thoughts on the matter?